crates/zed/src/languages/erlang/highlights.scm

```diff
@@ -0,0 +1,229 @@
+;; Copyright (c) Facebook, Inc. and its affiliates.
+;;
+;; Licensed under the Apache License, Version 2.0 (the "License");
+;; you may not use this file except in compliance with the License.
+;; You may obtain a copy of the License at
+;;
+;;     http://www.apache.org/licenses/LICENSE-2.0
+;;
+;; Unless required by applicable law or agreed to in writing, software
+;; distributed under the License is distributed on an "AS IS" BASIS,
+;; WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+;; See the License for the specific language governing permissions and
+;; limitations under the License.
+;; ---------------------------------------------------------------------
+
+;; Based initially on the contents of https://github.com/WhatsApp/tree-sitter-erlang/issues/2 by @Wilfred
+;; and https://github.com/the-mikedavis/tree-sitter-erlang/blob/main/queries/highlights.scm
+;;
+;; The tests are also based on those in
+;; https://github.com/the-mikedavis/tree-sitter-erlang/tree/main/test/highlight
+;;
+
+
+;; First match wins in this file
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Attributes
+
+;; module attribute
+(module_attribute
+  name: (atom) @module)
+
+;; behaviour
+(behaviour_attribute name: (atom) @module)
+
+;; export
+
+;; Import attribute
+(import_attribute
+    module: (atom) @module)
+
+;; export_type
+
+;; optional_callbacks
+
+;; compile
+(compile_options_attribute
+    options: (tuple
+      expr: (atom)
+      expr: (list
+        exprs: (binary_op_expr
+          lhs: (atom)
+          rhs: (integer)))))
+
+;; file attribute
+
+;; record
+(record_decl name: (atom) @type)
+(record_decl name: (macro_call_expr name: (var) @constant))
+(record_field name: (atom) @property)
+
+;; type alias
+
+;; opaque
+
+;; Spec attribute
+(spec fun: (atom) @function)
+(spec
+  module: (module name: (atom) @module)
+  fun: (atom) @function)
+
+;; callback
+(callback fun: (atom) @function)
+
+;; wild attribute
+(wild_attribute name: (attr_name name: (atom) @keyword))
```

Comment:
I found a bit of weird highlighting for `wild_attribute` node. It got highlighted as an `atom` (string.special.symbol) like screenshot below.

<img width="497" alt="Screenshot 2567-02-01 at 21 45 38" src="https://github.com/zed-industries/zed/assets/484530/f423973a-5d85-440f-b548-c63ceac63422">

You can see that `-dialyzer` should be highlight as same as `-record` and friends. After moving the `wild_attribute` node to under `atom` at line 229, it works. 

I'm not sure if it's the right fix. My tree-sitter knowledge is limited.

Reply:
Submit a fix at https://github.com/codeadict/zed/pull/1. 🙇

Reply:
I think the problem here is that dialyzer is not defined as a keyword and acts as an atom

Reply:
I would not put primitives so high in the file and rather move the wild attributes to override them. I have done that and here is the result:
![Screenshot 2024-02-01 at 11 29 54 AM](https://github.com/zed-industries/zed/assets/168440/eccb35f0-ebdc-40f7-b75e-093a3a157fc3)